conformance/conformance.proto

```diff
@@ -70,8 +71,15 @@ enum TestCategory {
   JSON_IGNORE_UNKNOWN_PARSING_TEST = 3;
   // Test jspb wire format. Google internal only. Opensource testees just skip it.
   JSPB_TEST = 4;
+  // Test text format. Google internal only currently. Opensource doesn't 
+  // have any test with this category for now. For cpp java and python,
+  // testee can already dual with this type.
```

Comment:
Do you mean deal?

Reply:
Test text format. For cpp, java and python, testees can already deal with this type. Testees of other languages can simply skip it.

Reply:
Done.